lib/linguist/language.rb

```diff
@@ -91,6 +92,8 @@ def self.create(attributes = {})
       Linguist::Strategy::Modeline,
       Linguist::Shebang,
       Linguist::Strategy::Filename,
+      Linguist::Strategy::Extension,
+      Linguist::Strategy::Filename,
```

Comment:
@pchaigno - looks like we've got this strategy in here twice. I'm assuming the second time is unintentional?